cli/popper/gha.py

```diff
@@ -2,8 +2,9 @@
 from builtins import dict, str
 import hcl
 import os
-import popper.scm as scm
 import popper.utils as pu
+import popper.scm as scm
+import git
```

Comment:
would it be possible to hide `git` within the `scm` module? Or alternatively, completely get rid of the `scm.py` file?

Reply:
@ivotron Getting rid of `scm.py` file completely is not possible i guess, as some functions in it like `get_user()` and `get_name()` etc. does not have direct API's in GitPython. So, one thing we can do is that keep the `scm.py` file and replace raw git commands with GitPython API's in there.